packages/shared/src/components/onboarding/ThemeOnboarding.tsx

```diff
@@ -35,16 +29,10 @@ function ThemeOnboarding({ onClose }: OnboardingStepProps): ReactElement {
             ))}
           </Container>
           <Modal.Footer justify={Justify.Between}>
-            <Button
-              className="btn-tertiary"
-              onClick={onStepBackward(previousStep)}
-            >
-              {index === 1 ? 'Close' : 'Back'}
+            <Button className="btn-tertiary" onClick={previousStep}>
+              {activeStepIndex === 1 ? 'Close' : 'Back'}
```

Comment:
If this button will bring the user back to the introduction, with the removal of `onClose`, then we can just keep the label as `Back`.

Reply:
Great catch